components/financial_assistance/app/views/financial_assistance/applications/_mec_check_results.html.erb

```diff
@@ -2,10 +2,20 @@
   <div class="border-bottom-1px"></div>
     <div class="er-success bg-success">
         <div class="text-success pl-1 pr-1">
-            <p>                    
-                <b><%= l10n("faa.mc_success") %></b>
+            <p>                   
+              <% 
+                medicaid_short_url = l10n("faa.mc_success_link")
+                medicaid_agency_url = EnrollRegistry[:enroll_app].setting(:medicaid_agency_url).item
+                medicaid_agency_phone = EnrollRegistry[:enroll_app].setting(:medicaid_agency_phone).item
+              %>
+              <b><%= l10n("faa.mc_success_existing", medicaid_agency_phone: link_to(medicaid_agency_phone, "tel:#{medicaid_agency_phone}", target: '_blank')) %></b>
+              <%= l10n("faa.mc_success_enroll", medicaid_agency_url: link_to(medicaid_short_url, medicaid_agency_url, target: '_blank')) %>
```

Comment:
Perhaps we can make faa.mc_success_enroll, faa.mc_success_phone into a single translation